src/Mod/Points/Gui/Workbench.cpp

```diff
@@ -57,6 +57,9 @@ Gui::ToolBarItem* Workbench::setupToolBars() const
     *pnt << "Points_Import"
          << "Points_Export"
          << "Separator"
+         << "Points_Convert"
+         << "Points_Structure"
+         << "Points_Merge"

```

Comment:
Does this look correct?

Reply:
Looks right to me - it compiles fine, and the icons display.